pkg/cmd/run.go

```diff
@@ -260,7 +261,9 @@ func RunMicroshift(cfg *config.Config) error {
 
 	select {
 	case <-ready:
-		klog.InfoS("MICROSHIFT READY", "since-start", time.Since(microshiftStart))
+		startRec.LogMicroshiftReady(time.Now())
```

Comment:
We could call `time.Now()` inside the function

Reply:
done